lib/public/views/Runs/ActiveColumns/runsActiveColumns.js

```diff
@@ -114,6 +115,12 @@ export const runsActiveColumns = {
         format: (timestamp) => formatTimestamp(timestamp, false),
         exportFormat: (timestamp) => formatTimestamp(timestamp),
     },
+    triggerValue: {
+        name: 'Trg Value',
+        visible: true,
+        size: 'w-5 f6 w-wrapped',
+        filter: triggerValueFilter,
```

Comment:
Missing `format` which should display `-` for missing values.
This will be important to have for previous runs

Reply:
Fixed in next commit